web/index.html

```diff
@@ -1,12 +1,26 @@
 <html>
 	<head>
+		<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
 		<title>My first three.js app</title>
-		<style>
+		<style type="text/css">
 			body { margin: 0; }
 			canvas { width: 100%; height: 100% }
+			#myButtonID {
+				position: fixed;
```

Comment:
Fix indentation issues

Reply:
Updated indentation of the entire file using Atom.